### ISSUE TITLE
DInput is unresponsive after using DS4Windows

### ISSUE THREAD

**Issue by Khaztaroth** [Ryochan7/DS4Windows#436]
I was testing whether or not I could have a profile use XInput and another one DInput, I'm aware that for that I need to uncheck the "Hide DS4 Controller" option, making the Xinput profile have double inputs. It turns out that when doing this, my DS4 controller **_is_** recognized as a DInput device though it doesn't register **_any_** button presses, so when I go back to my DInput only profile I can't do anything and changing back on the fly to Xinput does work, what's more, if I stop DS4Windows the DInput device still doesn't register button presses until I turn it off and back on.

TL:DR: Using Xinput at all blocks off DInput forcing me to turn the controller off and back on in order to use it as a Dinput device

**Comment by Ryochan7** [comment]
Unfortunately, that is an unavoidable problem when using a Bluetooth connection for the DS4 when using DS4Windows. That problem does not exist when using either a USB or a Sony Wireless Adapter connection. The problem really stems from the mode change that is done in the program so that extra data (touchpad, gyro, accel) will be sent from the controller and so force feedback will work. The change cannot be undone without disconnecting the controller from Windows.

**Comment by Khaztaroth** [comment]
Darn it, well at least I know now. To be honest, games have been kinda wonky anyways using DInput so I better stick to Xinput until there's a better way. Thanks for the answer
